cmd/aws-cloud-controller-manager/main.go

```diff
@@ -240,7 +285,7 @@ func startRouteController(ctx *cloudcontrollerconfig.CompletedConfig, cloud clou
 	}
 
 	// failure: more than one cidr and dual stack is not enabled
-	if len(clusterCIDRs) > 1 && !utilfeature.DefaultFeatureGate.Enabled(features.IPv6DualStack) {
+	if len(clusterCIDRs) > 1 && !utilfeature.DefaultFeatureGate.Enabled(app.IPv6DualStack) {
```

Comment:
Weird that the feature gate definition is here. We should maybe move it to https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/controller-manager/pkg/features/kube_features.go

Reply:
Specifcally referring to `features.IPv6DualStack`

Reply:
Can you open a PR to k/k to move that? The constant declaration is currently here https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/cloud-provider/app/core.go#L41-L45

Reply:
Sorry. That is something I missed. I will open an PR on K/K to address it.

Reply:
https://github.com/kubernetes/kubernetes/pull/96872
It might have to wait after v1.20 release. As of now, we have to use k8s.io/cloud-provider/app

Reply:
Will certainly have to wait, but I would make a case for cherry picking this back to v1.20 once it opens.

Reply:
Thanks @cici37!

Reply:
thanks Cici!